sources/packages/api/src/database/entities/base.model.ts

```diff
@@ -0,0 +1,22 @@
+import { CreateDateColumn, UpdateDateColumn } from 'typeorm';
+import { ColumnNames } from '../constant';
+
+/**
+ * The base model class which includes basic timestamp columns and other common activities
+ */
+export abstract class BaseModel {
+  /**
+   * Time Columns
+  */
+  @CreateDateColumn({
+    name: ColumnNames.CreateTimestamp,
+    nullable: true,
```

Comment:
Just to double-check, we are allowing NULL here because the columns on DB were created using the default as "now()" and the date will be assigned in DB, right?

Reply:
FIXED